tools/osbuilder/igvm-builder/igvm_builder.sh

```diff
@@ -38,13 +38,16 @@ load_config_distro()
 
 DISTRO="azure-linux"
 MODE="build"
+DEBUG="false"
 
-while getopts ":o:s:iu" OPTIONS; do
+while getopts ":o:s:d:iu" OPTIONS; do
 	case "${OPTIONS}" in
 		o ) OUT_DIR=$OPTARG ;;
 		s ) SVN=$OPTARG ;;
+		d ) DEBUG=$OPTARG ;;
 		i ) MODE="install" ;;
 		u ) MODE="uninstall" ;;
```

Comment:
Can you make the below change? Otherwise the inconsistency of `MODE` is confusing`.
```suggestion
		d ) MODE=$OPTARG ;;
		i ) OP="install" ;;
		u ) OP="uninstall" ;;
```